### ISSUE TITLE
Refactoring CSS

### ISSUE THREAD

**Issue by WilsonMonkey** [Canbanizers/Canban#103]
refactor to get consistent styles all over the page.
includes:
- positioning
- fonts
- sizes
- colors
- naming of buttons etc.

**Comment by ghost** [comment]
input fields underscored
button styling
sizes
positioning
colors (e,g, errormessages)
show scrollbar if nessecarry
menuitems
headings

**Comment by janz93** [comment]
will be done by saturday afternoon

**Comment by janz93** [comment]
I commit all my changes into the branch css_refactoring 
if you don't make anz comments to the changes 
i will merge them on sunday evening into the master.

**Comment by WilsonMonkey** [comment]
okay, we will reply till tomorrow afternoon.

**Comment by WilsonMonkey** [comment]
please create a pull request when your branch is read to merge. we will review the pull request and merge it afterwards

**Comment by janz93** [comment]
yes on sunday evening i will do the pull request and merge the branch into the master

**Comment by ghost** [comment]
no, pleace dont merge it to the master on your own. we will do that after a review
